### PR TITLE
Segmentation and intensity feature extraction

### PR DESCRIPTION
**Pull request by gouthamr321**
This is the corresponding code for the workflow to analyze the calcium signaling data as a case study that I discussed on friday.

I talked to Peyton today and he has some updated guidelines for how the participants can use colorizer for visualization, but implementation might take a little longer on my end. But before then, I still wanted to get your eyes on this code

Please see the [readme](https://github.com/AllenCell/PairUPWorkshop/blob/segmentation_and_intensity_feature_extraction/segmentation_and_intensity_feature_extraction/README.md) for instructions on how to reproduce

### CODE REVIEW

**Review by vianamp — APPROVED** [review]

**Comment by gouthamr321** [comment]
Ok this should be ready for review... I will likely reorganize this code before the workshop, but not planning on making any more major....any code structure feedback appreciated :) 

new changes here are:
1.  using bioio instead instead of aicsimage
2. zarr conversion for data
3. colorizer and web volume viewer visualization

See the [readme](https://github.com/AllenCell/PairUPWorkshop/blob/segmentation_and_intensity_feature_extraction/segmentation_and_intensity/README.md)  for instructions to reproduce.

**Comment by ShrimpCryptid** [comment]
@gouthamr321 can you rename the `viewer` directory and replace it with the `timelapse-feature-explorer` folder I sent you?

File: segmentation_and_intensity/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# starts a local server 
+python run_local_server.py 8080
+
+# On your browser, go to http://localhost:8080
+
+Copy the address of your collection file of interest and click the timelapse-feature-explorer directory to open the viewer and upload that address in the load data tab
+
+```
+
+## Data visualizatioin using web volume viewer
```
Comment (ShrimpCryptid) [inline segmentation_and_intensity/README.md]:
```suggestion
## Data visualization using web volume viewer
```

File: segmentation_and_intensity/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+We can also use the [Web-volume viewer](https://github.com/allen-cell-animated/volume-viewer) to visualize the results. A neccesary step here is converting the data into the OME-Zarr format
```
Comment (ShrimpCryptid) [inline segmentation_and_intensity/README.md]:
```suggestion
We can also use the [3D Volume Viewer](https://github.com/allen-cell-animated/volume-viewer) to visualize the results. A necessary step here is converting the data into the OME-Zarr format
```

File: segmentation_and_intensity/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+```
+# Converts timelapse data into the OME-Zarr standard
+python convert_to_zarr.py --input_dir input_dir --output_dir output_converted_movies_path
+
+#  similarly start a local server 
+python run_local_server.py 8080
+
+Copy the address of the Zarr file of interest and click the 3d-volume-viewer directory to open the viewer and upload that address in the load data tab
+
+```
```
Comment (ShrimpCryptid) [inline segmentation_and_intensity/README.md]:
````suggestion
```

Copy the address of the Zarr file of interest and click the 3d-volume-viewer directory to open the viewer and upload that address in the load data tab
````

File: segmentation_and_intensity/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## Training a custom cellpose model(Optional)
+
+Cellpose has many standard models that you can apply. This is often a great place to start to get a first pass segmentation. 
+
+However, if these are not generalizable, you can also train a custom model. We recomend using the cellpose GUI for creating annotations for a subset of data and exporting the annotations as png masks.
```
Comment (ShrimpCryptid) [inline segmentation_and_intensity/README.md:74]: Nit: Only these sentences have punctuation?

**Review by ShrimpCryptid — APPROVED** [review]
Just tested and the new timelapse-feature-explorer link works great, thank you! This looks good though I left some suggestions.

**Pull request merged**